roster/urls.py

```diff
@@ -3,4 +3,5 @@
 
 urlpatterns = [
     path('bosses', views.bosses),
+    path('bosses2', views.bosses2),
```

Comment:
I propose for paths use something like 'roster/bosses_selection' and 'roster/players_selection'

Reply:
Same to views functions names, something more adequate to what this function does